libraries/standard/json/cbmc/sources/skipGeneric.c

```diff
@@ -0,0 +1,33 @@
+#include "skipGeneric.h"
+
+/**
+ * See skipGeneric.h for docs
+ *
+ * Advance buffer index beyond some minimum value.
+ */
+static bool_ skipGeneric( const char * buf,
+                          size_t * start,
+                          size_t max,
+                          size_t min )
+{
+    bool_ ret = nondet_bool() ? true : false;
+
+    if( ret == true )
+    {
+        size_t x;
+        __CPROVER_assume( x >= min );
+        __CPROVER_assume( x <= max );
```

Comment:
If `min < max`, this will become `assume(0)`, and block further verification

Reply:
I thought this yielded some value x such that min <= x <= max.  You're saying that's not the case?

Reply:
Given the guard on line 38, this should be OK. Would it make sense to make `min <= max` an assert?

Reply:
The `min > max` is one of the false cases, which we want to exercise in addition to the true case.